readme.md

```diff
@@ -742,6 +742,7 @@
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
 - [Digital Humanities](https://github.com/dh-tech/awesome-digital-humanities#readme) - Software for humanities scholars using quantitative or computational methods.
 - [Lucid Dreams](https://github.com/IAmCoder/awesome-lucid-dreams#readme) - A dream where one becomes aware they are dreaming.
+- [Transgender Resources](https://github.com/cvyl/awesome-transgender#readme) - List of LGBTQ+ resources focussed on transgender individuals.
```

Comment:
```suggestion
- [Transgender](https://github.com/cvyl/awesome-transgender#readme) - Someone whose gender identity differs from their assigned birth sex.
```

I would go with this, but you can change it to a different definition if you want.